storage-api/src/java/org/apache/hadoop/hive/ql/exec/vector/VectorizedRowBatch.java

```diff
@@ -359,4 +361,62 @@ public void ensureSize(int rows) {
       cols[i].ensureSize(rows, false);
     }
```

Comment:
Should we call updateSelected(rows) here as well?
According to getMaxSize() method, selected array should be able to hold the max number of rows.

Reply:
Yeah, that is a good idea.